tag/feature.py

```diff
@@ -52,11 +52,28 @@ class Feature(object):
     """
 
     def __init__(self, data):
-        fields = data.split('\t')
-        assert len(fields) == 9
+        # Core values
+        self._seqid = None
+        self._source = None
+        self._type = None
+        self._range = None
+        self._score = None
+        self._strand = None
+        self._phase = None
+        self._attrs = dict()
```

Comment:
Looked into "constructor overloading" with `@classmethod` decorator, but for now settled on using `data=None` to build a skeleton feature.